packages/suite/src/actions/wallet/coinmarketCommonActions.ts

```diff
@@ -91,3 +94,14 @@ export const verifyAddress = (path: string, address: string) => async (
         );
     }
 };
+
+export const submitRequestForm = (tradeForm: BuyTradeFormResponse) => async (
+    dispatch: Dispatch,
+    getState: GetState,
+) => {
+    const { device } = getState().suite;
+    if (device) {
```

Comment:
```suggestion
    if (device && !device.remember) {
```
better safe than sorry :)

Reply:
It is checked inside rememberDevice but I added it there as you suggest it. Also, I added a check which will force remember device only in the web version, in the destop version we do not leave the Suite.